server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -79,10 +81,12 @@ export class TenantManager {
             if (isNetworkError(error)) {
                 if (error.code === 403 && !tenantKeys.key2){
                     // Delete key from cache when there is a 403 error and no key2 to validate
-                    Lumberjack.info(`Error with tenant key 1 token verification while key 2 is not present. Deleting key from cache.`, lumberProperties);
-                    await this.deleteKeyFromCache(tenantId).catch((err) => {
-                        Lumberjack.error(`Error deleting keys from the cache.`, lumberProperties, err);
-                    });
+                    if (bypassCache && this.isCacheEnabled) {
```

Comment:
It looks like we call `bypassCache` as `disableCache` in other locations in this file - if that's the case, should we adopt a single name for that type of parameter?

Reply:
Changed all to bypassCache to avoid confusion with isCacheEnabled